src/framework/MOM_diag_mediator.F90

```diff
@@ -4303,31 +4302,18 @@ end subroutine downsample_mask_3d
 
 !> Fakes a register of a diagnostic to find out if an obsolete
 !! parameter appears in the diag_table.
-logical function found_in_diagtable(diag, varName, newVarName)
+logical function found_in_diagtable(diag, varName)
   type(diag_ctrl),            intent(in) :: diag       !< A structure used to control diagnostics.
   character(len=*),           intent(in) :: varName    !< The obsolete diagnostic name
-  character(len=*), optional, intent(in) :: newVarName !< The valid name of this diagnostic
   ! Local
   integer :: handle ! Integer handle returned from diag_manager
 
   ! We use register_static_field_fms() instead of register_static_field() so
```

Comment:
This comment reminds me we should create an option to pass to register() that avoids logging the diagnostic so we can then also list static diagnostics in the list available diagnostics.